app/services/provider_setup.rb

```diff
@@ -27,12 +27,23 @@ def next_relationship_pending
   end
 
   def relationships_pending
-    auth = ProviderAuthorisation.new(actor: @provider_user)
-    auth.training_provider_relationships_that_actor_can_manage_organisations_for.select do |relationship|
+    manageable_relationships.select do |relationship|
       (relationship.setup_at.blank? || relationship.invalid?) && open_course_for_relationship?(relationship)
     end
   end
 
+private
+
+  def manageable_relationships
+    auth = ProviderAuthorisation.new(actor: @provider_user)
+
+    if FeatureFlag.active?(:accredited_provider_setting_permissions)
```

Comment:
This effects more than just the DSA success page now.

We also use the method in the relationships setup controller but we're changing that too, so I think it's fine. Might be worth another sanity check though

Reply:
Checked and we use in the main provider interface controller to trigger a setup/Setup/DSA and the navbar. I think all sounds find to me 👍